tests/vmi_multus_test.go

```diff
@@ -499,29 +499,35 @@ var _ = Describe("[Serial]Multus", func() {
 					linuxBridgeNetwork,
 				}
 
-				ep1Ip := "1.0.0.10/24"
-				ep2Ip := "1.0.0.11/24"
-				ep1IpV6 := "fe80::ce3d:82ff:fe52:24c0/64"
-				ep2IpV6 := "fe80::ce3d:82ff:fe52:24c1/64"
-				userdata := fmt.Sprintf(`#!/bin/bash
-                    echo "fedora" |passwd fedora --stdin
-                    setenforce 0
-                    ip link add ep1 type veth peer name ep2
-                    ip addr add %s dev ep1
-	                ip addr add %s dev ep2
-	                ip addr add %s dev ep1
-	                ip addr add %s dev ep2
-                    mkdir -p /usr/local/bin
-                    curl %s > /usr/local/bin/qemu-ga
-                    chmod +x /usr/local/bin/qemu-ga
-		    curl %s > /lib64/libpixman-1.so.0
-                    systemd-run --unit=guestagent /usr/local/bin/qemu-ga
-                `, ep1Ip, ep2Ip, ep1IpV6, ep2IpV6, tests.GetUrl(tests.GuestAgentHttpUrl), tests.GetUrl(tests.PixmanUrl))
-				agentVMI := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskFedora), userdata)
+				networkDataStruct, err := libnet.DefaultCloudInitNetworkData()
```

Comment:
We do not really need the default here. (e.g. the namserver is of no interest, so is the gateway and the IPv6 address on `eth0`. It would be nicer to see the whole config here.